community/fzf/APKBUILD

```diff
@@ -47,13 +48,18 @@ tmux() {
 	install -Dm0755 bin/fzf-tmux "$subpkgdir"/usr/bin/fzf-tmux
 }
 
-extras() {
-        description="additional scripts like shell completion and keybindings"
+_do_shell() {
+	description="additional scripts for $1 like shell completion and keybindings"
```

Comment:
You probably want to use `pkgdesc` instead of `description` here.

Besides: You want to add an `install_if` to make sure that the subpackage is installed automatically if the corresponding shell is installed. The completion subpackages of the `community/docker` subpackage provide such an `install_if` rule.

Reply:
fixed the description and added the install_if directives.